lib/vagrant-service-manager/command.rb

```diff
@@ -5,6 +5,7 @@ module Vagrant
   module ServiceManager
     DOCKER_PATH = '/home/vagrant/.docker'
     SUPPORTED_SERVICES = ['docker', 'openshift', 'kubernetes']
+    SCCLI_SERVICES = ["openshift", "kubernetes", "k8s"]
```

Comment:
Unless we have fix for it in `sccli` lets carry the patch in VSM, later we remove it.